MDANSE_GUI/Src/MDANSE_GUI/Tabs/Visualisers/DataPlotter.py

```diff
@@ -68,14 +68,16 @@ def __init__(self, *args, unit_lookup=None, **kwargs):
             unit_lookup=self._unit_lookup,
         )
         self._selection_viewer.setModel(self._model)
+        for col_num in range(4, 10):
+            self._selection_viewer.hideColumn(col_num)
 
     @Slot(object)
     def add_dataset(self, dataset: SingleDataset):
         if not dataset._valid:
             return
         self._model.add_dataset(dataset)
         self._selection_viewer.resizeColumnsToContents()
-        for col_num in range(4, 9):
+        for col_num in range(4, 10):
```

Comment:
Is there particular meaning to `4, 10`? Should this be a constant?

Reply:
This just hides specific columns in the PlottingContext table.